### PR TITLE
Added genetic_algoritm and candle_core

### PR DESCRIPTION
**Pull request by fstylermiller**

### CODE REVIEW

**Comment by fstylermiller** [comment]
Fix the lockfile dude

File: src/main.rs [hunk 1]
```diff
@@ -2,8 +2,9 @@
 #![cfg_attr(not(feature = "dev"), windows_subsystem = "windows")]
 
 use bevy::prelude::*;
-use you_are_bugs::AppPlugin;
+use you_are_bugs::{genetics::*, AppPlugin};
 
-fn main() -> AppExit {
-    App::new().add_plugins(AppPlugin).run()
+fn main() {
+    // App::new().add_plugins(AppPlugin).run()
```
Comment (fstylermiller) [inline src/main.rs:8]: wtf is this, fix it

File: Cargo.toml [hunk 2]
```diff
@@ -6,6 +6,8 @@ edition = "2021"
 
 [dependencies]
 bevy = { version = "0.14", features = ["wayland"] }
+candle-core = { git = "https://github.com/huggingface/candle.git", version = "0.6.0" }
+genetic_algorithm = { version = "0.7.1" }
```
Comment (fstylermiller) [inline Cargo.toml:10]: Adding dependencies? What are you, as JS dev?

File: src/main.rs (hunk 1 above)
Comment (corman44) [reply src/main.rs:8]: rubbish

**Pull request closed without merging**